### PR TITLE
MLPAB-850 Add change links for donor details and lpa type

### PR DESCRIPTION
**Pull request by hawx**
I've also tried to come up with a more generic way of redirecting back to the correct page

### CODE REVIEW

File: web/template/choose_attorneys_address.gohtml [hunk 1]
```diff
@@ -71,7 +71,7 @@
 
             {{ if gt (len .Lpa.ActorAddresses) 0 }}
               <p class="govuk-body">
-                <a href="{{ link .App .App.Paths.UseExistingAddress }}?type={{ .App.ActorTypes.Attorney }}&subjectId={{ .Attorney.ID }}&from={{ .App.Page }}" class="govuk-link govuk-link--no-visited-state">
+                <a href="{{ link .App .App.Paths.UseExistingAddress }}?type={{ .App.ActorTypes.Attorney }}&subjectId={{ .Attorney.ID }}&return={{ .App.Page }}" class="govuk-link govuk-link--no-visited-state">
```
Comment (hawx) [inline web/template/choose_attorneys_address.gohtml:74]: As I'd decided to use `from` I had to change these to stop them clashing

File: app/internal/page/donor/remove_person_to_notify.go [hunk 2]
```diff
@@ -42,32 +42,21 @@ func RemovePersonToNotify(logger Logger, tmpl template.Template, lpaStore LpaSto
 			data.Form = readRemovePersonToNotifyForm(r)
 			data.Errors = data.Form.Validate()
 
-			if data.Form.RemovePersonToNotify == "yes" && data.Errors.None() {
-				lpa.PeopleToNotify.Delete(attorney)
```
Comment (acsauk) [inline app/internal/page/donor/remove_person_to_notify.go:46]: Looks like some copy paste var naming has come over from remove attorneys here (although not as part of this ticket)

**Review by acsauk — COMMENTED** [review]
Changes all good - one old var name to be updated for clarity

**Review by acsauk — APPROVED** [review]
:shipit:

**Pull request merged**